dom-events/buttons.js

```diff
@@ -0,0 +1,24 @@
+function handleClick(event) {
+  console.log('button clicked');
+  console.log(event);
+  console.dir(event);
```

Comment:
the second call in each of the functions should be the `event.target` property. and should be console logs instead of dir.